tests/fixture_update_check/mod.ts

```diff
@@ -0,0 +1,10 @@
+import { updateCheck } from "$fresh/src/server/update_check.ts";
+
+// deno-lint-ignore require-await
+async function getLatestVersion() {
+  console.log("fetching latest version");
+  return Deno.env.get("LATEST_VERSION") ?? "1.2.0";
```

Comment:
Are we going to have to update this every time fresh releases a new version? Feels super unfortunate.

Reply:
No, this file is only used for the tests. The tests don't care in particular what the version is and are independent of the real Fresh version number.

Reply:
I wonder if `99.99.999` isn't clearer in terms of intent. Up to you.

Reply:
Good point, that avoids any confusion in the first place, will change that.